core/primer.md

```diff
@@ -33,7 +34,130 @@ associating those declarations with application endpoints. As a result, the
+centralized registry, individual teams who want to discover, query, produce and
+consume events from endpoints have a single point of truth to refer to. In
+addition, xRegistry allows specifying extensions and metadata that can further
+describe events on more detailed levels. As an example, it is possible to define
+an event, and mark specific metadata that are required. Due to its rich metadata
+model, metadata can also be expressed in human language, making it easier for
+developers, data-analysts, or even business analysts to understand.
+
+### Vendor-agnostic
+
+xRegistry provides a specification to define, query, group, version and enforce
+schemas for systems. Multiple schema registry solutions exist already,
+including [Confluent Schema Registry](https://docs.confluent.io/platform/current/schema-registry/index.html), [Azure Event Hubs Registry](https://learn.microsoft.com/en-us/azure/event-hubs/schema-registry-overview)
+or [Apicurio Registry](https://www.apicur.io/registry/), [AWS Glue Schema Registry](https://docs.aws.amazon.com/glue/latest/dg/schema-registry.html)
+and more.
+
+These schema registries are broker- and therefore vendor-specific and even
+protocol-specific, which causes friction when an event travels through multiple
+brokers. In such cases, clients have to deal with the implementation differences
+across registries, for example when validating event structures, hindering
+interoperability and, therefore, complicating integration scenarios. xRegistry
+aims to address this with an agnostic specification, providing a common base
+across these offerings.
+
+### Versioning
+
+As systems evolve, the endpoints that emit events may change, as may the events
+themselves. Changes can sometimes break consumers making compatibility
+strategies essential to indicate to consumers how they may expect events or
+endpoints to evolve. xRegistry allows indicating a registry’s compatibility
+strategy which is enforced when making changes to the registry, avoiding
+mistakes. The specifications further also enables marking endpoints as
+deprecated, possibly providing an alternative endpoint, making these easier to
+discover for consumers. Finally, event schemas are also version aware, storing
+multiple versions of an event’s data schema.
+
+### Schema validation
+
+The registry contains granular definitions for events defining their schemas,
+which serve as a contract between publishers and consumers. This doesn’t only
+apply to the message payload, but also to the message metadata, allowing
+granular control of required fields on the metadata level. The registry does not
+only serve as a definition of the required schemas, but can also be used to
+enforce it. A registry service could use the registry to perform validation
+on-publish, rejecting any events that don’t adhere to the schema for that
+context, avoiding the problem of poison messages.
+
+### Payload reduction
```

Comment:
This is just saying that payloads can have a URL to a schema doc instead of embedding it, right?

Reply:
Yes that's the intent, was it unclear?

Reply:
no, it's fine - I just wanted to make sure I wasn't missing some other point. I'm slow sometimes :-)